lib/client/order.rb

```diff
@@ -10,7 +10,14 @@ def get(id, params = {})
           response = client.get(uri({ order_id: id }), params)
           order = response.body
         rescue Faraday::Error => e
-          raise BigcommerceApiError.new('get order', { order_id: id }, e)
+          raise BigcommerceApiError.new(
+            e.response[:status],
+            'get order',
+            "Failed to get order #{id}",
+            id,
+            { order_id: id },
```

Comment:
We're already passing the order ID. We don't need to include it in the data as well. This data object should only have the errors from the BigCommerce response

Reply:
@mrogers-5s resolved